### PR TITLE
Update Fable 2.2 & Fable.Core v3

### PR DESCRIPTION
**Pull request by forki**
this includes https://github.com/SAFE-Stack/SAFE-BookStore/pull/388

### CODE REVIEW

**Comment by forki** [comment]
@MangelMaxime I wish we could get rid of the prereleases in the deps file

**Comment by forki** [comment]
image

where is the ReactServer!?

**Comment by alfonsogarciacaro** [comment]
Fable.ReactServer (just remove Helpers) :)

**Comment by alfonsogarciacaro** [comment]
To remove `prerelease` we need to release stable versions, to release stable version Maxime wants the SAFE Bookstore to work with the prerelease versions... so it's a chicken-and-egg situation.

**Comment by forki** [comment]
Ok good. I'm fine with using bookstore as trial balloon

Alfonso Garcia-Caro <notifications@github.com> schrieb am Mo., 15. Apr.
2019, 10:16:

**Comment by MangelMaxime** [comment]
@forki I wanted to test porting a SAFE application before releasing the stable version. In order to check that everything is OK and see if any changes were needed. For example, we discovered that now we need to have separates groups between Client and Server. So we can guide people, in the migration process.

If this branch is working then we can release the stable version :)

**Comment by forki** [comment]
ok looks good now

File: src/Client/pages/WishList.fs [hunk 1]
```diff
@@ -115,19 +114,19 @@ let update (msg:Msg) model : Model * Cmd<Msg> =
...
+                                (model.NewBookModel.NewBook :: model.WishList.Books)
                                 |> List.sortBy (fun b -> b.Title) }
 
                 let submodel,cmd = NewBook.init()
                 { model with WishList = wishList; NewBookModel = submodel; ErrorMsg = None },
                     Cmd.batch [
                         Cmd.map NewBookMsg cmd
-                        Cmd.ofPromise postWishList (model.Token,wishList) FetchedWishList FetchError
-                    ]            
+                        Cmd.OfPromise.either postWishList (model.Token,wishList) FetchedWishList FetchError
```
Comment (forki) [inline src/Client/pages/WishList.fs]: urgs!? what's that?

File: src/Client/pages/NewBook.fs [hunk 2]
```diff
@@ -58,9 +58,9 @@ let update (msg:Msg) model : Model*Cmd<Msg> =
                 TitleErrorText = model.NewBook.ValidateTitle()
                 AuthorsErrorText = model.NewBook.ValidateAuthors()
                 LinkErrorText = model.NewBook.ValidateLink() }
-        validated, 
+        validated,
             if model.NewBook.Validate() then
-                Cmd.ofMsg AddBook
+                Cmd.OfFunc.result AddBook
```
Comment (forki) [inline src/Client/pages/NewBook.fs]: urgs!?

File: src/Client/pages/WishList.fs (hunk 1 above)
Comment (MangelMaxime) [reply src/Client/pages/WishList.fs]: A long discussion lead to this changes: https://github.com/elmish/elmish/issues/166

File: src/Client/pages/NewBook.fs (hunk 2 above)
Comment (MangelMaxime) [reply src/Client/pages/NewBook.fs]: A long discussion leads to this changes: https://github.com/elmish/elmish/issues/166

**Pull request merged**